test/test-app/vendor/kubeflow/core/tf-job-operator.libsonnet

```diff
@@ -450,6 +453,7 @@
           ],
           resources: [
             "tfjobs",
+            "tfjobs/status",
```

Comment:
Just to understand, Will this have any problems with  v1beta1 crd where sub resource is not enabled?

Reply:
This just allows the operator to update tfjobs/status, so it does not affect the v1beta1 crd.